src/app/feed/feed.component.ts

```diff
@@ -16,8 +16,8 @@ import { Title } from "@angular/platform-browser";
 export class FeedComponent implements OnInit, OnDestroy, AfterViewChecked {
   static GLOBAL_TAB = "Global";
   static FOLLOWING_TAB = "Following";
-  static MARKET_TAB = "Market";
-  static TABS = [FeedComponent.GLOBAL_TAB, FeedComponent.FOLLOWING_TAB];
+  static SHOWCASE_TAB = "💎Showcase💎";
```

Comment:
Does this look weird without spaces between emoji and text?

Reply:
Without spaces:
![image](https://user-images.githubusercontent.com/81658138/127203060-c3815a40-d2bf-4250-9375-2ecc57f24150.png)

With spaces:
![image](https://user-images.githubusercontent.com/81658138/127203175-8f3b9223-5d8d-49ad-bbe1-f9cac928a767.png)

spaces feels a bit too spaced out IMO

Reply:
Interesting, I definitely prefer the spaces

Reply:
I might also prefer just one diamond at the beginning or end